MapView/Map/RMBackgroundOperationWorker.m

```diff
@@ -0,0 +1,77 @@
+//
+//  RMBackgroundOperationWorker.m
+//  MapView
+//
+//  Created by Natalia Paula Osiecka on 22/09/16.
+//  Copyright (c) 2014 Ordnance Survey. All rights reserved.
+//
+
+#import "RMBackgroundOperationWorker.h"
+
+@interface RMBackgroundOperationWorker()
+
+@property(nonatomic, readonly) NSArray<id> *observerIds;
```

Comment:
I think it adds value here. In Swift it wouldn't, because it'd be obvious that any object is accepted. In objc however it may mean that the developer didn't implement generics(most of this module is not written with generics). Because of this, the array can have particular type of data, but it's not marked which one. By adding `<id>` here, the reader is sure that it can have any type of objects and (s)he does not have to browse `.m` to search for the type (worrying that the type is just not added here).